system-test/integration_test.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build integration && go1.7
```

Comment:
Curious - what is the difference between these two lines?

```
//go:build integration && go1.7
// +build integration,go1.7
```

They look similar - to the point where I'm thinking "hmm, maybe one of them is redundant?"...

Reply:
I think the minimal needed is `// +build` and things will still be working.

Go 1.16 will fail to build if only `//go:build` is used.

Go 1.17 and onward, it is recommended to use `//go:build`.

`gofmt` automatically adds `//go:build`, which is a safe thing to do.

But now that you've pointed it out, in integration_test.sh, the test is always built with Go 1.17, so having only the newer version would suffice.

Let me try that out and if the tests pass, I'll use the newer syntax, unless there's a reason we want to still run the tests in Go 1.16.

Reply:
Oh, and some context where I read these things: [tracking bug](https://github.com/golang/go/issues/41184) and [transition plan](https://golang.org/s/go-build-design#transition).

Reply:
Yep, things still work using just the newer syntax. Thanks for pointing it out.